python/semantic_kernel/connectors/ai/open_ai/services/open_ai_config_base.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from logging import Logger
+from typing import Any, Dict, Optional
+
+from openai import AsyncOpenAI
+
+from semantic_kernel.connectors.ai.open_ai.services.open_ai_handler import (
+    OpenAIHandler,
+)
+from semantic_kernel.connectors.ai.open_ai.services.open_ai_model_types import (
+    OpenAIModelTypes,
+)
+
+
+class OpenAIConfigBase(OpenAIHandler):
+    def __init__(
```

Comment:
This code looks good. Is it necessary to have any further error handling here? e.g., an invalid API key is provided, or an invalid model is provided?

Reply:
I think those are runtime checks, since you might have proxies in between that do different key formats etc, or like AzureOpenAI that can use AAD auth instead of a key. Instantiating a class is unlikely to do a actual check since that would be a async call and that shouldn't happen in a init. We could decide to do some checks with pydantic, similar to what happens in the AIServiceClientBase, where the model_id is at least 1 character.